setup.py

```diff
@@ -30,8 +30,8 @@
     ]
 
     internal_requirements = [
-        'jobmon>=2.2.1,<2.3',
-        'db_queries>=24.4.0,<25',
+        'jobmon[ihme]==3.0.4rc1',
```

Comment:
Why pin it?

Reply:
Because I don't know when and how things will change and we just need something that works reliably.